opentreemap/treemap/audit.py

```diff
@@ -1319,6 +1319,8 @@ def _unit_format(self, value):
                 value = value.area
         elif isinstance(value, datetime):
             value = dformat(value, settings.SHORT_DATE_FORMAT)
+        elif isinstance(value, list):
+            value = '(%s)' % ', '.join(value) if value else 'none'
```

Comment:
You should probably wrap `'none'` in `_(...)` for translation, with a translation comment for context.